modules/i3s/src/i3s-webscene-loader.ts

```diff
@@ -0,0 +1,34 @@
+import type {LoaderWithParser, LoaderOptions, LoaderContext} from '@loaders.gl/loader-utils';
+import type {I3SWebScene} from './types';
+
+import {parseI3SWebscene} from './lib/parsers/parse-i3s-webscene';
+
+// __VERSION__ is injected by babel-plugin-version-inline
+// @ts-ignore TS2304: Cannot find name '__VERSION__'.
+
```

Comment:
Remove empty line as comment on 7th line is for 9th line.

Reply:
Done